myia/ir/node.py

```diff
@@ -157,6 +176,11 @@ def add_debug(self, **kwargs):
             for k, v in kwargs.items():
                 setattr(self.debug, k, v)
 
+    def _clone(self, old, objmap):
```

Comment:
Confusing name for what that method does.

Reply:
I'm open to better suggestions.

Reply:
`_copy_fields`?

Reply:
I'll go with `_copy_fields`